openwisp_monitoring/notifications/migrations/0003_default_permissions.py

```diff
@@ -0,0 +1,49 @@
+from django.db import migrations
+from django.contrib.auth.models import Permission
+from django.contrib.auth.management import create_permissions
+
+
+def create_default_groups(apps, schema_editor):
+    group = apps.get_model('openwisp_users', 'group')
+
+    # To populate all the permissions
+    for app_config in apps.get_app_configs():
+        app_config.models_module = True
+        create_permissions(app_config, apps=apps, verbosity=0)
+        app_config.models_module = None
+   
+    operator = group.objects.filter(name='Operator')
+    if operator.count() == 0:
+        operator = group.objects.create(name='Operator')
+    else:
+        operator = operator[0]
+    permissions = [ Permission.objects.get(content_type__app_label='notifications',
+                                           codename='view_notification').pk
+    ]
```

Comment:
indentation is odd, please make it linear, eg:

```python
permissions = [
    line1,
    line2,
]
```

Reply:
But, I should break them right?

Reply:
```python
permissions = [
    Permission.objects.get(content_type__app_label='notifications',
                                         codename='view_notification').pk
]
``